src/print.ts

```diff
@@ -63,6 +63,19 @@ function print(type: string) {
 
     if (vscode.workspace.getConfiguration("markdown.extension.print", doc.uri).get<boolean>("absoluteImgPath")) {
         body = body.replace(/(<img[^>]+src=")([^"]+)("[^>]*>)/g, function (match, p1, p2, p3) { // Match '<img...src="..."...>'
+            if (vscode.workspace.getConfiguration("markdown.extension.print", doc.uri).get<boolean>("imgToBase64")) {
+                try {
+                    let root;
+                    const imgPath = (root = vscode.workspace.getWorkspaceFolder(doc.uri))
+                        ? path.join(root.uri.fsPath, p2)
+                        : path.join(path.dirname(doc.uri.fsPath), p2);
```

Comment:
How about changing the `fixHref` function to return an `Uri` so that we can reuse it here (`imgPath = fixHref(...).fsPath`).